### PR TITLE
Enhancement/8847 RRM GA tracking

### PR DESCRIPTION
**Pull request by ankitrox**
## Summary

Addresses issue:

- #8847

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 7.4.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [x] Reassess the implementation with the IB.
- [x] Ensure no unrelated changes are included.
- [x] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [x] Ensure the PR has the correct target branch.
- [x] Double-check that the PR is okay to be merged.
- [x] Ensure the corresponding issue has a ZenHub release assigned.
- [x] Add a changelog message to the issue.

### CODE REVIEW

**Review by techanvil — CHANGES_REQUESTED** [review]
Thanks @ankitrox, this is off to a good start. I've left a few comments for your consideration.

Furthermore, please can you add a QAB?

**Comment by ankitrox** [comment]
Thank you @techanvil for reviewing the PR.

I've addressed the feedback and sending it your way for another review.

File: assets/js/modules/reader-revenue-manager/components/common/PublicationOnboardingStateNotice.test.js [hunk 1]
```diff
@@ -120,6 +135,17 @@ describe( 'PublicationOnboardingStateNotice', () => {
 			expect( container.firstChild ).toHaveClass(
 				'googlesitekit-publication-onboarding-state-notice'
 			);
+
+			// eslint-disable-next-line require-await
+			await act( async () => {
+				fireEvent.click( getByText( ctaText ) );
+			} );
```
Comment (techanvil) [inline assets/js/modules/reader-revenue-manager/components/common/PublicationOnboardingStateNotice.test.js:142]: Hmm, I didn't notice this the first time round, but it doesn't look like this needs to be async either.

More importantly though - as per my previous [comment](https://github.com/google/site-kit-wp/pull/9179#pullrequestreview-2240750196), the issue still needs a QAB. Please can you add one?

```suggestion
			act( () => {
				fireEvent.click( getByText( ctaText ) );
			} );
```

**Review by techanvil — CHANGES_REQUESTED** [review]
Thanks @ankitrox! This is looking good, just one last minor point to address in the PR - I would have simply applies this myself, but the issue also still needs a QAB so I am sending it back your way.

**Comment by ankitrox** [comment]
@techanvil Thank you. I've updated the QAB and removed async await mentioned.

**Review by techanvil — APPROVED** [review]
LGTM, nice one @ankitrox!

**Pull request merged**